build/configs/rtl8721csm/rdp/defconfig

```diff
@@ -0,0 +1,1114 @@
+# CONFIG_DOWNLOAD_IMAGE is not set
+
+#
+# Customize Header Files
+#
+# CONFIG_ARCH_STDINT_H is not set
+# CONFIG_ARCH_STDBOOL_H is not set
+# CONFIG_ARCH_MATH_H is not set
+# CONFIG_ARCH_FLOAT_H is not set
+# CONFIG_ARCH_STDARG_H is not set
+CONFIG_ARCH_HAVE_CUSTOMOPT=y
+CONFIG_DEBUG_NOOPT=y
+# CONFIG_DEBUG_CUSTOMOPT is not set
+# CONFIG_DEBUG_FULLOPT is not set
+
+#
+# Chip Selection
+#
+CONFIG_ARCH_ARM=y
+# CONFIG_ARCH_XTENSA is not set
+CONFIG_ARCH="arm"
+CONFIG_ARCH_CHIP_AMEBAD=y
+CONFIG_ARCH_CHIP="amebad"
+CONFIG_RTL8721D=y
+# CONFIG_FPGA is not set
+CONFIG_CHIP_A_CUT=y
+# CONFIG_CHIP_B_CUT is not set
+
+#
+# ARM Options
+#
+# CONFIG_ARCH_CORTEXM3 is not set
+# CONFIG_ARCH_CORTEXM4 is not set
+CONFIG_ARCH_CORTEXM33=y
+CONFIG_ARCH_FAMILY="armv8-m"
+CONFIG_ARCH_HAVE_CMNVECTOR=y
+CONFIG_ARM_CMNVECTOR=y
+# CONFIG_ARCH_HAVE_FPU is not set
+# CONFIG_ARCH_HAVE_DPFPU is not set
+# CONFIG_ARCH_FPU is not set
+# CONFIG_ARM_HAVE_MPU is not set
+# CONFIG_ARM_MPU is not set
+# CONFIG_ARMV8M_MPU is not set
+# CONFIG_ARMV8M_MPU_NREGIONS is not set
+
+#
+# Exception stack options
+#
+# CONFIG_ARCH_HAVE_DABORTSTACK is not set
+
+#
+# ARMV8M Configuration Options
+#
+CONFIG_ARMV8M_HAVE_ICACHE=y
+CONFIG_ARMV8M_HAVE_DCACHE=y
+CONFIG_ARMV8M_USEBASEPRI=y
+CONFIG_ARMV8M_TRUSTZONE=y
+CONFIG_ARMV8M_ICACHE=y
+CONFIG_ARMV8M_DCACHE=y
+# CONFIG_ARMV8M_HAVE_ITCM is not set
+# CONFIG_ARMV8M_HAVE_DTCM is not set
+# CONFIG_ARMV8M_TOOLCHAIN_BUILDROOT is not set
+# CONFIG_ARMV8M_TOOLCHAIN_CODEREDL is not set
+# CONFIG_ARMV8M_TOOLCHAIN_CODESOURCERYL is not set
+CONFIG_ARMV8M_TOOLCHAIN_GNU_EABIL=y
+# CONFIG_ARMV8M_TOOLCHAIN_CLANGL is not set
+# CONFIG_ARMV8M_HAVE_STACKCHECK is not set
+# CONFIG_ARMV8M_ITMSYSLOG is not set
+
+#
+# Architecture Options
+#
+# CONFIG_ARCH_NOINTC is not set
+# CONFIG_ARCH_VECNOTIRQ is not set
+# CONFIG_ARCH_DMA is not set
+CONFIG_ARCH_HAVE_IRQPRIO=y
+# CONFIG_ARCH_L2CACHE is not set
+# CONFIG_ARCH_HAVE_COHERENT_DCACHE is not set
+# CONFIG_ARCH_HAVE_ADDRENV is not set
+# CONFIG_ARCH_NEED_ADDRENV_MAPPING is not set
+# CONFIG_ARCH_HAVE_VFORK is not set
+# CONFIG_ARCH_HAVE_MMU is not set
+CONFIG_ARCH_HAVE_MPU=y
+# CONFIG_ARCH_NAND_HWECC is not set
+# CONFIG_ARCH_HAVE_EXTCLK is not set
+# CONFIG_ARCH_HAVE_POWEROFF is not set
+# CONFIG_ARCH_HAVE_RESET is not set
+# CONFIG_ARCH_USE_MPU is not set
+CONFIG_ARCH_IRQPRIO=y
+CONFIG_ARCH_STACKDUMP=y
+# CONFIG_ENDIAN_BIG is not set
+# CONFIG_ARCH_IDLE_CUSTOM is not set
+# CONFIG_ARCH_HAVE_RAMFUNCS is not set
+CONFIG_ARCH_HAVE_RAMVECTORS=y
+# CONFIG_ARCH_RAMVECTORS is not set
+# CONFIG_SUPPRESS_INTERRUPTS is not set
+# CONFIG_SUPPRESS_TIMER_INTS is not set
+
+#
+# Board Settings
+#
+CONFIG_BOARD_LOOPSPERMSEC=16717
+# CONFIG_ARCH_CALIBRATION is not set
+
+#
+# Interrupt options
+#
+# CONFIG_ARCH_HAVE_INTERRUPTSTACK is not set
+# CONFIG_ARCH_HAVE_NESTED_INTERRUPT is not set
+# CONFIG_ARCH_HAVE_HIPRI_INTERRUPT is not set
+
+#
+# Boot Memory Configuration
+#
+CONFIG_RAM_REGIONx_START="0x0,0x0,"
```

Comment:
Is it okay not to set the RAM address?

Reply:
this is not set even in the default hello and tc configs for Realtek